<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="1" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-02-01T10:44:59 IST" name="Suite" finished-at="2022-02-01T10:47:28 IST" duration-ms="148280">
    <groups>
      <group name="Sanity">
        <method signature="LogInTests.logInTest()[pri:0, instance:com.myshop.testcases.LogInTests@4466af20]" name="logInTest" class="com.myshop.testcases.LogInTests"/>
        <method signature="LogInTests.logInTest1(java.lang.String,java.lang.String)[pri:0, instance:com.myshop.testcases.LogInTests@4466af20]" name="logInTest1" class="com.myshop.testcases.LogInTests"/>
      </group> <!-- Sanity -->
      <group name="Smoke">
        <method signature="LogInTests.logInTest()[pri:0, instance:com.myshop.testcases.LogInTests@4466af20]" name="logInTest" class="com.myshop.testcases.LogInTests"/>
        <method signature="LogInTests.logInTest1(java.lang.String,java.lang.String)[pri:0, instance:com.myshop.testcases.LogInTests@4466af20]" name="logInTest1" class="com.myshop.testcases.LogInTests"/>
      </group> <!-- Smoke -->
    </groups>
    <test started-at="2022-02-01T10:44:59 IST" name="TestChrome" finished-at="2022-02-01T10:47:28 IST" duration-ms="148280">
      <class name="com.myshop.testcases.LogInTests">
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.myshop.testcases.LogInTests@4466af20]" started-at="2022-02-01T10:45:00 IST" name="setup" finished-at="2022-02-01T10:45:52 IST" duration-ms="52541" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="logInTest()[pri:0, instance:com.myshop.testcases.LogInTests@4466af20]" started-at="2022-02-01T10:45:52 IST" name="logInTest" finished-at="2022-02-01T10:46:13 IST" duration-ms="20543" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logInTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.myshop.testcases.LogInTests@4466af20]" started-at="2022-02-01T10:46:13 IST" name="tearDown" finished-at="2022-02-01T10:46:14 IST" duration-ms="1391" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.myshop.testcases.LogInTests@4466af20]" started-at="2022-02-01T10:46:17 IST" name="setup" finished-at="2022-02-01T10:46:33 IST" duration-ms="16422" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="logInTest1(java.lang.String,java.lang.String)[pri:0, instance:com.myshop.testcases.LogInTests@4466af20]" started-at="2022-02-01T10:46:33 IST" name="logInTest1" data-provider="credentials" finished-at="2022-02-01T10:46:44 IST" duration-ms="10639" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin@xyz.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[admin@123]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at com.myshop.testcases.LogInTests.logInTest1(LogInTests.java:64)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logInTest1 -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.myshop.testcases.LogInTests@4466af20]" started-at="2022-02-01T10:46:44 IST" name="tearDown" finished-at="2022-02-01T10:46:45 IST" duration-ms="441" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.myshop.testcases.LogInTests@4466af20]" started-at="2022-02-01T10:46:45 IST" name="setup" finished-at="2022-02-01T10:46:55 IST" duration-ms="10851" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="logInTest1(java.lang.String,java.lang.String)[pri:0, instance:com.myshop.testcases.LogInTests@4466af20]" started-at="2022-02-01T10:46:55 IST" name="logInTest1" data-provider="credentials" finished-at="2022-02-01T10:47:07 IST" duration-ms="11496" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[dom1@abc.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[dom@345]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at com.myshop.testcases.LogInTests.logInTest1(LogInTests.java:64)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logInTest1 -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.myshop.testcases.LogInTests@4466af20]" started-at="2022-02-01T10:47:07 IST" name="tearDown" finished-at="2022-02-01T10:47:07 IST" duration-ms="368" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.myshop.testcases.LogInTests@4466af20]" started-at="2022-02-01T10:47:07 IST" name="setup" finished-at="2022-02-01T10:47:17 IST" duration-ms="10040" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="logInTest1(java.lang.String,java.lang.String)[pri:0, instance:com.myshop.testcases.LogInTests@4466af20]" started-at="2022-02-01T10:47:17 IST" name="logInTest1" data-provider="credentials" finished-at="2022-02-01T10:47:27 IST" duration-ms="9756" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[shubha11@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[abc@123]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at com.myshop.testcases.LogInTests.logInTest1(LogInTests.java:64)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logInTest1 -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.myshop.testcases.LogInTests@4466af20]" started-at="2022-02-01T10:47:27 IST" name="tearDown" finished-at="2022-02-01T10:47:28 IST" duration-ms="413" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.myshop.testcases.LogInTests -->
    </test> <!-- TestChrome -->
  </suite> <!-- Suite -->
</testng-results>
